Generator.Domain/Choice.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+
+namespace Generator.Domain
+{
+    public class Choice
+    {
+        public Guid PictureA { get; set; }
```

Comment:
czy to musi wszystko miec publiczne settery? Moze ustawic bez settera i ustawiac to w konstruktorze?